src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -65,129 +122,228 @@ export const WorkspaceList = ({ registeredUseCases$ }: WorkspaceListProps) => {
     [application, http]
   );
 
-  const searchResult = useMemo(() => {
-    if (queryInput) {
-      const normalizedQuery = queryInput.toLowerCase();
-      const result = workspaceList.filter((item) => {
-        return (
-          item.id.toLowerCase().indexOf(normalizedQuery) > -1 ||
-          item.name.toLowerCase().indexOf(normalizedQuery) > -1
-        );
-      });
-      return result;
-    }
-    return workspaceList;
-  }, [workspaceList, queryInput]);
+  const formatDate = function (dateString: string) {
+    const months = [
+      'Jan',
+      'Feb',
+      'Mar',
+      'Apr',
+      'May',
+      'Jun',
+      'Jul',
+      'Aug',
+      'Sep',
+      'Oct',
+      'Nov',
+      'Dec',
+    ];
+
+    const date = new Date(dateString);
+
+    const month = months[date.getUTCMonth()];
+    const day = String(date.getUTCDate()).padStart(2, '0');
+    const year = date.getUTCFullYear();
+
+    const hours = String(date.getUTCHours()).padStart(2, '0');
+    const minutes = String(date.getUTCMinutes()).padStart(2, '0');
+    const seconds = String(date.getUTCSeconds()).padStart(2, '0');
+    const milliseconds = String(date.getUTCMilliseconds()).padStart(3, '0');
+
+    return `${month} ${day},${year}@${hours}:${minutes}:${seconds}.${milliseconds}`;
+  };
```

Comment:
can we use [moment.js](https://momentjs.com/) to achieve this?

Reply:
Thanks for you review, [moment.js](https://momentjs.com/) is really useful, 
![Screenshot 2024-08-07 at 18 20 01](https://github.com/user-attachments/assets/6a3d174b-6046-4611-b3bc-a8ca67ebb2f9)
but I'm afraid there is not the expected format shown in figma

Reply:
can you try with format `'MMM DD[,]YYYY [@] HH:mm:ss.SSS'` reference https://momentjs.com/docs/#/displaying/

Reply:
Yeah, this works, thank you!

Reply:
![image](https://github.com/user-attachments/assets/ae70ae83-10de-4b8a-b1c7-549fa22e7292)
Shall we use the date format setting in the advanced settings?

Reply:
Thanks for your comment, already updated.